File: Mark/Lists/Day-2/remove duplicate/remove-duplicate.py
class Solution:
    def removeDuplicates(self, nums: list[int]) -> int:
        for num in nums:
            if nums.count(num) > 1:
                nums.remove(num)
            else:
                continue
        return len(nums)
        

solution = Solution()
output = solution.removeDuplicates([1,1,1,2,2,3,3,4,5])
print(output)

File: MargretEdeh/Day02/removeDuplicate.js
/**
 * @param {number[]} nums
 * @return {number}
 */
 var removeDuplicates = function(nums) {
    let j =0;
    for (let i =0; i< nums.length; i++){
        if(nums[i] == nums[i+1] )continue;
nums[j++]= nums[i]
    }
    return j;
};
const numbers = [0, 0, 1, 1, 2, 2, 3, 3, 4]
console.log(removeDuplicates(numbers))